packages/terra-brand-footer/src/BrandFooter.scss

```diff
@@ -7,19 +7,18 @@
     background-color: var(--terra-brand-footer-nav-background-color, #153d70);
     color: var(--terra-brand-footer-nav-color, #8baaca);
 
+    display: flex;
```

Comment:
Can you point to the screenshots / test examples that display this behavior?

Reply:
Could you clarify what you mean?

Reply:
In your deployment link, which examples can I look at to see the flex style being applied? In the screenshots, it appears the navigational links are displaying horizontal and I though we were introducing an option to show a vertical list display with these changes.

Reply:
Ah. https://terra-framework-deploye-pr-202.herokuapp.com/#/tests/terra-brand-footer/brand-footer/vertical-nav-brand-footer demonstrates the vertical layout, but I'll add an example to the documentation page.